### PR TITLE
Operating Unit Validity Date

### PR DESCRIPTION
**Pull request by jsierp**

### CODE REVIEW

File: operating_unit_validity_date/static/src/js/relational_fields.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                this.value.res_ids.forEach(id => {
+                    this._createOperatingUnitValidtyWarning(id);
+                    this.$el
+                        .find(`[data-id=${id}]`)
+                        .prepend(this.operatingUnitValidityWarnings[id]);
+                });
+            }
+        },
+        _updateOperatingUnitValidityWarning: function(ids) {
+            var self = this;
```
Comment (StefanWiselka-Solvti) [inline operating_unit_validity_date/static/src/js/relational_fields.js:78]: In many2one you added extra if for argument `id`
Why we would not need it here for `ids` ? (or it's forgotten)?

```
if (id === undefined) {
    self.$operatingUnitValidityWarning.hide();
} else {
    this._rpc({
```

**Review by StefanWiselka-Solvti — APPROVED** [review]
Just one question.

File: operating_unit_validity_date/static/src/js/relational_fields.js (hunk 1 above)
Comment (jsierp) [reply operating_unit_validity_date/static/src/js/relational_fields.js:78]: Checking ids is not necessary - if the field is not set ids variable is an empty list and thus doesn't cause any error. But we could actually add an if for this case to save us one rpc request.

Comment (StefanWiselka-Solvti) [reply operating_unit_validity_date/static/src/js/relational_fields.js:78]: That's' what i thought to do. thx

**Pull request merged**